app/models/group.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# OpenSCAP group
+class Group < ApplicationRecord
+  has_ancestry(primary_key_format: /\A[\w\-]+(\/[\w\-]+)*\z/)
+
+  belongs_to :benchmark, class_name: 'Xccdf::Benchmark'
```

Comment:
I feel like this should be handled via a polymorphic relation mixed with ancestry. However, I'm not sure if that would be as efficient as this is right now. We can keep it as it is for now but let's talk about it.

Reply:
True, that's a good point... Should Rule have a polymorphic relation since it can belong to Benchmark or Group?

Reply:
Let's deal with this later, it might worth to keep this due to efficiency. All groups belong to a benchmark, so this kinda makes sense and we might need this looked up and then it's faster to do it directly than by going through parent groups...

Reply:
That was my thought when I was working on this, that all groups belong to a benchmark, and same with Rule